<ui version="4.0" >
 <class>lifeConfig</class>
 <widget class="QWidget" name="lifeConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>193</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="verticalCellsLabel" >
     <property name="text" >
      <string>Vertical cells number:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KIntSpinBox" name="verticalCells" >
     <property name="minimum" >
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="horizontalCellsLabel" >
     <property name="text" >
      <string>Horizontal cells number:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="KIntSpinBox" name="horizontalCells" >
     <property name="minimum" >
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="stepIntervalLabel" >
     <property name="text" >
      <string>One generation each:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="KIntSpinBox" name="stepInterval" >
     <property name="suffix" >
      <string> sec</string>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="maxGensNumberLabel" >
     <property name="text" >
      <string>Restart game each:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="KIntSpinBox" name="maxGensNumber" >
     <property name="suffix" >
      <string> gens</string>
     </property>
     <property name="minimum" >
      <number>10</number>
     </property>
     <property name="maximum" >
      <number>999999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
